PluginCore/DockPanelSuite/Docking/DockPanel.FocusManager.cs

```diff
@@ -416,13 +416,13 @@ internal void SetActiveContent()
                     return;
 
                 if (m_activeContent != null)
-                    m_activeContent.DockHandler.IsActivated = false;
+                    m_activeContent.DockHandler.SetIsActivated(false);
```

Comment:
Have you  compared the behavior with the latest DPS? Do they do the same or do they have the same problem? If the latter you may want to create a PR for them.